content/departments/engineering/index.md

```diff
@@ -10,7 +10,18 @@ The Engineering department at Sourcegraph consists of:
 
 _1-year vision:_ Sourcegraph is a code intelligence platform, which devs will use 10+ times daily to unblock themselves and code better. Sourcegraph will be the fastest, and often only, way for devs to find what they need because it integrates all kinds of code-related information and makes it accessible within seconds on the web, in the editor, in code review, in custom tooling, and anywhere else it's needed.
 
-See "[Engineering department update 2022-06](https://docs.google.com/document/d/1YezFhbIsH8YHRLRdwZ-qUK1Z4aogM04_8IqxJpUX-nY/edit#)" for our strategy, including making Cloud (managed instances) the preferred deployment method, focusing on the core workflow, counting dev time saved, and deeply integrating code-related concepts (code nav, code graph, code ownership, and more).
+Notes:
+
+- This vision is valid through 2023-09. That technically makes it a 15-month vision (not 12-month) from when we adopted it.
+- When we say devs will use it "10+ times daily", that's the high-level intent but not strict success criteria.
+  - We want to track and increase frequency of usage, and we want it to approach and/or exceed 10 times daily for as many devs as possible.
+  - If there are kinds or groups of devs who don't use Sourcegraph frequently (much less 10+ times daily), we should figure out why and see if we can make Sourcegraph more useful to them.
+  - For example, if a dev works at a company where only some of the code they work with is on Sourcegraph, then it's less likely they'll find Sourcegraph useful 10+ times daily. The most valuable first step is probably to figure out how to get all of their code on Sourcegraph.
+- What is the relationship between "10+ times daily" and "dev time saved"?
+  - We want users to use Sourcegraph 10+ times daily _because_ it saves them time.
+  - We don't want to artificially increase frequency of usage through annoying "growth hack" techniques that don't save devs time.
```

Comment:
Can you clarify what you define as annoying growth hacks?  Any of these on that list? 

* inviting users
* notifications
* @mentions
* emails
* integrations with messaging platforms